grafana/README.md

```diff
@@ -0,0 +1,26 @@
+# Grafana dashboards
+
+This directory holds different Grafana dashboard JSON models. For details about the Grafana dashboard JSON model, see: https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/view-dashboard-json-model/
+
+In case you want to import these dashboards to Grafana, you can do so by following these steps:
+https://grafana.com/docs/grafana/v9.0/dashboards/export-import/. It is worthwhile mentioning that importing dashboards into Grafana requires Editor rights as a Grafana user.
+
+## Monitoring architecture
+
+Polkadot introspector is currently deployed in Versi. Versi is a test network which runs both a relay chain and several parachains in a Kubernetes environment. 
```

Comment:
I don't think mentioning `Versi` is an important detail for users. We should just provide the steps for a successful deployment.